### ISSUE TITLE
Node hover scale

### ISSUE THREAD

**Issue by rctneil** [BALKANGraph/FamilyTreeJS#131]
Hi,

I wish to make a node scale up slightly when hovered. I've added the following to my CSS:

        svg.fhf .node {
            transition: all 0.2s ease;
            transform-origin: 50% 50%;

            &:hover {
                transform: scale(1.1);
            }
        }
        
It works, but node moves to one corner.  It seems as though the origin is elsewhere, but i'm unsure as to why.

**Comment by ZornitsaPesheva** [comment]
You can try this function: https://balkan.app/FamilyTreeJS/API/classes/FamilyTree#magnify
Here is an example:
```
family.onInit(() => {
    family.magnify(1, 1.25, true, null);
})
```

**Comment by rctneil** [comment]
Is it possible to do this on hover?

**Comment by ZornitsaPesheva** [comment]
We have created this code example for you:
https://code.balkan.app/familytree-js/change-the-size-on-hover#CSS

**Comment by rctneil** [comment]
Ah ok.  So it's not possible to have the entire node scale up? including all the content?   That's what I was hoping for.

I do appreciate all the help!

**Comment by ZornitsaPesheva** [comment]
Yes, it is not possible to see all nodes, but it will be possible in the new implementation.

**Comment by rctneil** [comment]
> Yes, it is not possible to see all nodes, but it will be possible in the new implementation.

I don't wish to see all the node.    I want the whole node to increase in size, not just the <rect>, I want the text to scale up on the hover as well.

**Comment by ZornitsaPesheva** [comment]
Sorry, I was confused by the other post of yours.
I don't know the answer of this question, but I will find it out.

**Comment by rctneil** [comment]
No problem! Thankyou!

**Comment by ZornitsaPesheva** [comment]
We have edited the code example:
https://code.balkan.app/familytree-js/change-the-size-on-hover#CSS
